src/main/java/com/blackducksoftware/integration/alert/provider/blackduck/descriptor/BlackDuckDescriptor.java

```diff
@@ -38,4 +40,8 @@ public BlackDuckDescriptor(final BlackDuckProviderDescriptorConfig providerDescr
         super(PROVIDER_NAME, providerDescriptorConfig, provider);
     }
 
+    @Override
+    public Set<String> getNotificationTypes() {
+        return getProvider().getNotificationTypes();
```

Comment:
We should have this method not be abstract and just call this line in the parent class